packages/node_modules/nav-frontend-hjelpetekst-style/src/hjelpetekst-style.less

```diff
@@ -1,226 +1,250 @@
-
-        &:before,
-        &:focus:after {
-            transform-origin: 50% 50%;
-            transform: translate(-50%, -50%) rotate(-45deg);
-            left: 50%;
-
-        }
-    }
-
-    .hjelpetekst__tooltip--under,
-    .hjelpetekst__tooltip--under-venstre {
-        left: 0;
-        top: 3rem;
-
-        &:before,
-        &:after {
-            top: 0;
-        }
-
-        &:focus:after {
-            border-top-color: @orangeFocus;
-            border-right-color: @orangeFocus;
-        }
-
-        &:before {
-            left: ~'calc(1rem + 4px)';
-        }
-    }
-
-    .hjelpetekst__tooltip--under-venstre {
-        left: auto;
-        right: -1.5rem;
-
-        &:before,
-        &:after {
-            left: auto;
-            right: 1.95rem;
-        }
-
-        &:focus:after {
-            right: ~'calc(1.95rem - 2px)';
-        }
-    }
-
-    .hjelpetekst__tooltip--hoyre {
-        left: 4rem;
-        top: -2rem;
-        bottom: auto;
-
-        &:before {
-            top: 2rem;
-        }
-
-        &:before,
-        &:focus:after {
-            left: 0;
-            transform: rotate(45deg);
-        }
-
-        &:focus:after {
-            top: 1.75rem;
-            left: 0;
-            border-left: 3px solid @orangeFocus;
-            border-bottom: 3px solid @orangeFocus;
-        }
-    }
-
-    .hjelpetekst__tooltip--venstre {
-        right: 3rem;
-        left: auto;
-        top: -2rem;
-        bottom: auto;
-
-        &:before {
-            top: 2rem;
-        }
-
-        &:before,
-        &:focus:after {
-            right: 0;
-            transform-origin: 100% 0;
-        }
-
-        &:focus:after {
-            top: 1.75rem;
-            right: 0;
-            border-right: 3px solid @orangeFocus;
-            border-bottom: 3px solid @orangeFocus;
-        }
-    }
-}
+@import (reference) '~nav-frontend-core/less/_variabler';
+@import (reference) '~nav-frontend-core/less/_mixins';
+@import (reference) '~nav-frontend-core/less/scaffolding';
+@import (reference) '~nav-frontend-lukknapp-style/src/lukknapp-style';
+@import (reference) '~nav-frontend-typografi-style/src/index';
+@import (reference) '~nav-frontend-lenker-style/src/lenker-mixins';
+
+@desktop: ~'only screen and (min-width: 480px)';
+
+.hjelpetekst {
+    position: relative;
+    padding-left: 1.25rem;
+    display: inline-block;
+}
+
+.hjelpetekst .lukknapp {
+    .lukknapp();
+    position: absolute;
+    top: 1rem;
+    right: 1rem;
+}
+
+.hjelpetekst__tekst:before {
+    content: '';
+    display: block;
+    width: 4rem;
+    height: 3rem;
+    float: right;
+}
+
+.hjelpetekst__apneknapp {
+    margin: 0;
+    padding: 0;
+    border: 0;
+    cursor: pointer;
+    background-color: transparent;
+
+    &:focus,
+    &:active,
+    &:hover {
+        outline: 0;
+        color: @white;
+    }
+
+    &:focus .hjelpetekst__anchor {
+        outline: 0;
+        box-shadow: 0 0 0 3px @orangeFocus;
```

Comment:
Hm, virker som du har laget denne PRen på en ganske mye eldre versjon av kodebasen. Alle disse fokus-markeringene skal f.eks. bruke `@fokusFarge` i stedet for `@orangeFocus` (Ref. #461). Desto viktigere at line endings på denne fila blir fikset slik at vi ser hva som faktisk blir endret.

Reply:
Jeg har oppdatert